### DISCUSSION TITLE
Use one instance of MPV at a time

### DISCUSSION THREAD

**Discussion by advice2020** [mpv-player/mpv#14408] [Q&A]
Linux Debian based distro
MPV  0.35.1

I apologize, I have seen this mentioned before, but the post was closed and was from 2015-2018.
I was wondering if by any chance this is now possible in 2024?
I am looking for a built in setting that allows only one instance of MPV to run at a time.

---------------------------------------------------------------------------------------------------------------------------------------

`ALREADY AWARE OF`
-I am aware that you can drag & drop content into MPV GUI using your mouse to achieve this, looking for what is mentioned above.
-I have not tried it yet, but I am aware that there is a 3rd party script for this called UMPV that was mentioned in that closed post, but was looking for something built in.

`USES / DESIRED FEATURES`
-Main use would be to be able to load up content from a 3rd party file manager into one instance of MPV and when you do so MPV window gets focus.

---------------------------------------------------------------------------------------------------------------------------------------
MPV has settings for everything, was just strange not to also see an option available for this as well as the UMPV plugin.

Thank you to anyone who reads this and for any information provided

**Comment by guidocella** [comment]
umpv is a first party script available at https://github.com/mpv-player/mpv/blob/master/TOOLS/umpv

**Reply by AuthenticSm1les** [reply]
could umpv be configured so that it plays the selected media directly instead of adding it to queue or is there a different tool that implements a similar behavior?

**Reply by guidocella** [reply]
Just remove `append-play` from umpv.

**Comment by advice2020** [comment]
@guidocella 
Thank you for your response.
As mentioned I knew of UMPV because I have seen others mention it.  I overlooked that it was provided with MPV itself which is nice, I had thought it was a 3rd party script.
Took me a little while to understand it, but I tried this and was able to get this to work.
I still have a couple of questions about it but this works great.
I think I understand more now why this approach of a separate script was used.

Thank You

**Reply by LouayHanouche** [reply]
hey , im looking to achieve the same thing , is there a way to have this functionality in uosc ?
